src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14642,6 +14651,35 @@ void WriteLoadComponentSummaryTables(EnergyPlusData &state)
                 CombineLoadCompResults(FacilityHeatCompLoadTables, FacilityZonesHeatCompLoadTables(iZone), mult);
             }
 
+            bool CoincidentFlag = false;
+            for (int SysSizIndex = 1; SysSizIndex <= state.dataSize->NumSysSizInput; ++SysSizIndex) {
+                if (state.dataSize->SysSizInput(SysSizIndex).SizingOption == DataSizing::Coincident) {
+                    CoincidentFlag = true;
+                }
+            }
```

Comment:
So if **any** system requests sizing, then they all report it this way?  Sorry if that's a silly question.

Reply:
@Myoldmopar Good point. Let me look at this issue further.

Reply:
@Myoldmopar I checked the code carefully and think CoincidentFlag is not necessary, because all variables of peak loads are overridden in the system level with coincident, while remain as is with noncoincident.

I am going to upload the change soon.